src/FlexForms/FlexFormsProcessor.php

```diff
@@ -73,4 +63,33 @@ public function getSupportedFileExtensions(): array
     {
         return ['xml'];
     }
+
+    private function processFile(File $file): void
+    {
+        $smartFileInfo = $file->getSmartFileInfo();
+
+        $domDocument = new DOMDocument();
+
+        $domDocument->formatOutput = true;
+
+        $domDocument->load($smartFileInfo->getRealPath());
```

Comment:
Imho you should use File->getContent instead of smartFileInfo->getContent or path because it not working with dry run well